DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/L1/test_l1_forced_speed.py

```diff
@@ -0,0 +1,166 @@
+import pytest
+import asyncio
+
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from dent_os_testbed.lib.ethtool.ethtool import Ethtool
+from dent_os_testbed.lib.onlp.onlp_system_info import OnlpSystemInfo
+
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_get_traffic_stats,
+    tgen_utils_setup_streams,
+    tgen_utils_start_traffic,
+    tgen_utils_stop_traffic,
+    tgen_utils_dev_groups_from_config,
+    tgen_utils_traffic_generator_connect,
+    tgen_utils_get_loss
+)
+
+
+pytestmark = [pytest.mark.suite_functional_l1,
+              pytest.mark.asyncio,
+              pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")]
+
+
+@pytest.mark.parametrize("speed , duplex",
+                         [
+                            # (10, "full"),
+                            # (10, "half"),
+                            # (100, "full"),
+                            # (100, "half")
+                            # (1000, "full"),
+                            # (1000, "half"),
+                            (10000, "full"),
+                         ])
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "speed, duplex",
    [# (10, "full"), (10, "half"),
     # (100, "full"), (100, "half"),
     # (1000, "full"), (1000, "half"),
     (10000, "full")],
)
```

Reply:
How are you going to test 10/100/1000 speed?

Reply:
Another issue is that running each test would take a long time. Consider testing both full and half duplex during a single test run (10 full+half, 100 full+half, 1000 full, 10_000 full)